libs/core/src/lib/dynamic-page/dynamic-page-header/header/dynamic-page-header.component.ts

```diff
@@ -48,6 +52,9 @@ export class DynamicPageHeaderComponent implements OnInit, AfterViewInit, OnDest
     @ContentChild(BreadcrumbComponent)
     _breadcrumbComponent: BreadcrumbComponent;
 
+    @ContentChild(FacetComponent)
+    _facetComponent: FacetComponent;
```

Comment:
Missing @hidden typedoc

Reply:
I don't see any usages of this content child component

Reply:
Removed